single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/svg/styles/iidm/TopologicalStyleProvider.java

```diff
@@ -36,19 +36,33 @@ public class TopologicalStyleProvider extends AbstractVoltageStyleProvider {
     private final Map<String, Integer> stylesIndices = new HashMap<>();
     private final Network network;
     private final SvgParameters svgParameters;
+    private boolean busesHighlightOnHover;
```

Comment:
```suggestion
    private boolean busHighlightOnHover;
```
Same here for plural